x-pack/docs/en/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -0,0 +1,133 @@
+[role="xpack"]
+[[security-api-grant-api-key]]
+=== Grant API key API
+++++
+<titleabbrev>Grant API keys</titleabbrev>
+++++
+
+Creates an API key on behalf of another user for access without requiring basic
+authentication.
+
+[[security-api-grant-api-key-request]]
+==== {api-request-title}
+
+`POST /_security/api_key/grant`
+
+[[security-api-grant-api-key-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have the `grant_api_key` cluster privilege.
+
+[[security-api-grant-api-key-desc]]
+==== {api-description-title}
+
+This API is similar to <<security-api-create-api-key>>, however it creates the
+API key for a user that is different than the user that runs the API.
+
```

Comment:
I think we should have an explanation of _why_ this API exists & why someone would use it .

```suggestion

The caller must have authentication credentials (either an access token, or a username and password) for the user on whose behalf the API key will be created. It is not possible to use this API to create an API key without that user's credentials.

This API is intended be used by applications that need to create and manage API keys for end users, but cannot guarantee that those users have permission to create API keys on their own behalf (see <<security-api-create-api-key-prereqs>>). 
```